ztp/ran-crd/site-config-crd.yaml

```diff
@@ -84,6 +84,13 @@ spec:
                         enum:
                           - standard
                           - sno
+                      numMasters:
```

Comment:
Do we need to expose this as an input in the CRD, or can PolicyGen set the value automatically based on either clusterType or the length of the nodes array?

Reply:
AI lets u define number of masters 3, 5, ...etc I guess we should follow the same